### PR TITLE
Fixes #25501 - Add timeout when sending reports from proxy

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Related change in installer: https://github.com/theforeman/puppet-foreman_proxy/pull/467

File: lib/smart_proxy_openscap/foreman_forwarder.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@ module Proxy::OpenSCAP
   class ForemanForwarder < Proxy::HttpRequest::ForemanRequest
     include ::Proxy::Log
 
-    def post_arf_report(cname, policy_id, date, data)
+    def post_arf_report(cname, policy_id, date, data, timeout)
```
Comment (ares) [inline lib/smart_proxy_openscap/foreman_forwarder.rb]: this is also used athttps://github.com/theforeman/smart_proxy_openscap/blob/master/lib/smart_proxy_openscap/openscap_import_api.rb#L15

either make timeout optional or change every call

File: lib/smart_proxy_openscap/foreman_forwarder.rb [hunk 2]
```diff
@@ -25,13 +25,14 @@ def upload_path(cname, policy_id, date)
       "/api/v2/compliance/arf_reports/#{cname}/#{policy_id}/#{date}"
     end
 
-    def send_request(path, body)
+    def send_request(path, body, timeout)
```
Comment (ares) [inline lib/smart_proxy_openscap/foreman_forwarder.rb]: this seems to be also used at https://github.com/theforeman/smart_proxy_openscap/blob/master/lib/smart_proxy_openscap/fetch_file.rb#L49 either make timeout optional or modify it everywhere

**Comment by xprazak2** [comment]
I added the params that were missing.

**Comment by xprazak2** [comment]
I updated the comment

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**